web/store/project/project_filter.store.ts

```diff
@@ -67,6 +72,20 @@ export class ProjectFilterStore implements IProjectFilterStore {
     return this.displayFilters[workspaceSlug];
   }
 
+  /**
+   * @description get project state applied display filter of the current workspace
+   * @returns {TProjectAppliedDisplayFilterKeys[] | undefined} // An array of keys of applied display filters
+   */
+  // TODO: Figure out a better approach for this
+  get currentWorkspaceAppliedDisplayFilters() {
+    const workspaceSlug = this.rootStore.app.router.workspaceSlug;
+    if (!workspaceSlug) return;
+    const displayFilters = this.displayFilters[workspaceSlug];
+    return Object.keys(displayFilters)
```

Comment:
you dont't have to call filter twice, you can return both conditions within a single filter

Reply:
Sure. Optimizing it.